src/lib/containers/widgets/RangeSlider.tsx

```diff
@@ -150,6 +150,7 @@ export const RangeSlider: React.FunctionComponent<RangeSliderProps> = ({doUpdate
 
   return (
     <div className="rangeSlider">
+      <div className="rangeSlider__values">{values[0]} - {values[1]}</div>
```

Comment:
I added this so that the user can immediately see what the values they are setting before hitting apply.
![image](https://user-images.githubusercontent.com/5190623/72080982-867b0c80-32b2-11ea-813c-47ccb8a5e87c.png)